src/slowmokit/methods/activation_functions.hpp

```diff
@@ -0,0 +1,77 @@
+/**
+ * @file methods/activation_functions.hpp
+ *
+ * Easy include to add non-linearity into a neural network.
+ */
+
+#ifndef ACTIVATION_FUNCTIONS_HPP
+#define ACTIVATION_FUNCTIONS_HPP
+#include "../core.hpp"
+template<class T>
+/**
+ * @brief To calculate sigmoid(x)
+ * @param x: Number whose sigmoid value is to be calculated
+ * @return a double value representing sigmoid(x)
+ */
+double sigmoid(double);
+
+/**
+<<<<<<< HEAD
+ * @param x {double x} - double value on which the function is applied.
+ *
+ * @param x {vector<double>} - vector containing 'double' values of x for
+ * softmax activation function implementation.
+ *
+ * @return {double value} - double value after putting x in the functions gets
+ * returned.
+=======
```

Comment:
```suggestion
```

Reply:
> 

@Ishwarendra sir the changes which you've asked to make, these are not visible in my vs code, those unwanted lines of code are already not there.

Reply:
i formatted all the source files using gitbash, since the clang error was there, but after commiting and pushing those changes, both the checks are coming wrong, how do i fix it?